app/src/components/BaseFilterNav.vue

```diff
@@ -63,6 +66,7 @@ export default defineComponent({
   name: 'BaseFilterNav',
   components: {},
   props: {
+    title: { type: String, required: false },
```

Comment:
I feel like the title should be required as to ensure the delineation between the two sections. Is there a use case where we'd rather not have it appear?

Reply:
agree with this

Reply:
@corydickson looks like there are scenrios where this would be used with BaseHeader and a title isn't required